docs/usage.rst

```diff
@@ -112,25 +112,6 @@ the **get-file-locations** command:
 
 This command will output URIs for all the files associated with the record ID 5500, using the HTTP protocol.
 
-**HTTPS protocol**
```

Comment:
It is good to remove this section, but then we may not be having any HTTPS note anywhere.  So you can add here:

```
Note that you can specify `--server https://opendata.cern.ch` if you would like to use the HTTPS protocol instead.
```

in those places where we spoke about HTTPS... In this way when people search the docs for HTTPS they would find at least a note on this.